@kiva/kv-components/vue/KvContentfulImg.vue

```diff
@@ -45,6 +45,11 @@
 			<!-- browser doesn't support webp or browser doesn't support picture element -->
 			<img
 				class="tw-max-w-full tw-max-h-full"
+				style="
+					width: inherit;
+					height: inherit;
+					object-fit: inherit;
```

Comment:
guessing we don't have tw classes for these.

Reply:
None that I could find